packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx

```diff
@@ -111,7 +112,26 @@ const SidebarNavigation: React.FC<SidebarNavigationProps> = () => {
           </SiteNavNestList>
         </SiteNavItem>
         <SiteNavItem>
-          <SiteNavAnchor to="/tokens">Tokens</SiteNavAnchor>
+          <SiteNavButton onClick={() => setTokensOpen(!tokensOpen)} isOpen={tokensOpen}>
+            Design Tokens
+            <SiteNavAnchorArrow isOpen={tokensOpen} />
+          </SiteNavButton>
+          <SiteNavNestList isOpen={tokensOpen}>
+            <SiteNavItem>
+              <SiteNavAnchor to={SidebarCategoryRoutes.TOKENS}>Token list</SiteNavAnchor>
+            </SiteNavItem>
+            <SiteNavItem>
+              <SiteNavAnchor to={`${SidebarCategoryRoutes.TOKENS}/how-to-compose-custom-ui-with-tokens`}>
+                How to compose custom UI with tokens
+              </SiteNavAnchor>
+            </SiteNavItem>
+            <SiteNavItem>
+              <SiteNavAnchor to={`${SidebarCategoryRoutes.TOKENS}/design-tokens-package`}>Tokens package</SiteNavAnchor>
```

Comment:
> Tokens package" 

Should we call this "Design Tokens Package" to match how we write design-tokens elsewhere?

Reply:
probably